tutorials/tree/dictionary/writeTree.cxx

```diff
@@ -0,0 +1,56 @@
+// Author: Alvaro Tolosa-Delgado CERN 07/2023
+// Author: Jorge Agramunt Ros IFIC(Valencia,Spain) 07/2023
+
+/*************************************************************************
+ * Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include <iostream>
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TTreeReader.h>
+#include <TTreeReaderValue.h>
+
+#include "data2Tree.hxx"
+
+void writeTree()
+{
+   std::unique_ptr<TFile> ofile{TFile::Open("testFile.root", "recreate")};
+   if (!ofile || ofile->IsZombie()) {
+      throw std::runtime_error("Could not open file testFile.root");
+   }
+
+   std::unique_ptr<TTree> myTree = std::make_unique<TTree>("myTree", "");
+   myDetectorData obj_for_branch1;
+   myTree->Branch("branch1.", &obj_for_branch1);
```

Comment:
I am still missing here a brief comment, perhaps 1 or 2 lines, explaining of the trailing dot (see [@pcanal's comment](https://github.com/root-project/root/pull/13205#discussion_r1265655686)).